scripts/build_android.sh

```diff
@@ -89,6 +89,7 @@ CMAKE_ARGS+=("-DUSE_LMDB=OFF")
 CMAKE_ARGS+=("-DUSE_LEVELDB=OFF")
 CMAKE_ARGS+=("-DUSE_MPI=OFF")
 CMAKE_ARGS+=("-DUSE_OPENMP=OFF")
+CMAKE_ARGS+=("-DUSE_FBGEMM=OFF")
```

Comment:
another option is to do this in master CMake directly? https://github.com/pytorch/pytorch/blob/master/CMakeLists.txt#L224
as we might need create other scripts for different type of mobile builds it probably makes sense to add this to a centralized place?

Reply:
Thanks.
I tried to update this pull request, but accidentally opened a new one with updated commit

https://github.com/pytorch/pytorch/pull/24104